### PR TITLE
Fix compilation and remove nphysics mentions

### PR DESCRIPTION
**Pull request by ThierryBerger**
- needed for new PR to pass their tests (https://github.com/dimforge/salva/pull/53)

### CODE REVIEW

File: src/object/fluid.rs [hunk 1]
```diff
@@ -165,13 +165,6 @@ impl Fluid {
         self.positions.len()
     }
 
-    /// Computes the AABB of this fluid.
-    #[cfg(feature = "nphysics")]
-    pub fn compute_aabb(&self, particle_radius: Real) -> ncollide::bounding_volume::AABB<Real> {
-        use ncollide::bounding_volume::{self, BoundingVolume};
-        bounding_volume::local_point_cloud_aabb(&self.positions).loosened(particle_radius)
-    }
-
```
Comment (ThierryBerger) [inline src/object/fluid.rs]: This should probably be kept but translated for parry ?

File: Cargo.toml [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 
 [profile.release]
 #lto = true
 #codegen-units = 1
 
-[profile.dev]
-# Use slightly better optimization by default, as examples otherwise seem laggy.
-opt-level = 1
-
 [patch.crates-io]
```
Comment (sebcrozet) [inline Cargo.toml]: Why don’t we keep this?

File: src/object/fluid.rs (hunk 1 above)
Comment (sebcrozet) [reply src/object/fluid.rs]: Agreed. The translation should look very similar. Parry’s point-cloud calculation function is in `bounding_volume::details::local_point_cloud_aabb`.

**Review by sebcrozet — APPROVED** [review]
Thanks!

File: Cargo.toml (hunk 2 above)
Comment (ThierryBerger) [reply Cargo.toml]: Ah my latest commit didn't go through

**Pull request merged**